mono/mini/mini-arm.c

```diff
@@ -7329,14 +7329,14 @@ mono_arch_set_breakpoint (MonoJitInfo *ji, guint8 *ip)
 	MonoDebugOptions *opt = mini_get_debug_options ();
 
 	if (ji->from_aot) {
-		SeqPointInfo *info = mono_arch_get_seq_point_info (mono_domain_get (), ji->code_start);
+		SeqPointInfo *info = (SeqPointInfo*)mono_arch_get_seq_point_info (mono_domain_get (), (guint8*)ji->code_start);
```

Comment:
Future or maybe present:
1. Change return type of `mono_arch_get_seq_point_info` to `SeqPointInfo*`.
2. Change last parameter of `mono_arch_get_seq_point_info` to `gpointer`. And then maybe just assign right back to local of same type (i.e. in case of math/sizeof(deref)).